### PR TITLE
Port Docker containers to Python3

### PR DESCRIPTION
**Pull request by daniviga**
Another step for https://github.com/gem/oq-engine/issues/2803 phase 1

### CODE REVIEW

File: oq-docker/Dockerfile.centos [hunk 1]
```diff
@@ -5,25 +5,29 @@ MAINTAINER Daniele Viganò <daniele@openquake.org>
 ARG oq_branch=master
 ARG tools_branch=master
 
-RUN yum -q -y install epel-release python curl && \
-    yum -q -y install python-pip && \
-    pip -q install -U pip wheel
+RUN yum -q -y install centos-release-scl curl
+# Using Python 3.6 here since Python 3.5 from SCL is too old
+# user should not care about what is running inside Docker
```
Comment (daniviga) [inline oq-docker/Dockerfile.centos:10]: RedHat SCL provides Python 3.5.1 which has a bug with relative imports thus breaks `openquakeplatform.utils`

File: oq-docker/Dockerfile.ubuntu [hunk 2]
```diff
@@ -5,32 +5,33 @@ MAINTAINER Daniele Viganò <daniele@openquake.org>
...
+RUN apt-get -qq update && \
+    apt-get -qq install -y python3 python3-pip curl && \
+    apt-get -qq clean all && \
+    pip3 -q install -U pip wheel
 
 RUN mkdir -p /opt/openquake/lib && \
     ln -rs /opt/openquake/lib /opt/openquake/lib64
 
-ENV PYTHONPATH /opt/openquake/lib/python2.7/site-packages/
+ENV PYTHONPATH /opt/openquake/lib/python3.5/site-packages/
```
Comment (hascar) [inline oq-docker/Dockerfile.ubuntu:16]: 3.5 or 3.6 ?

Comment (daniviga) [reply oq-docker/Dockerfile.ubuntu:16]: This is Ubuntu. Ubuntu Xenial has 3.5. The exception is CentOS

**Review by hascar — APPROVED** [review]
LGTM

**Pull request merged**